hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/endpoint/RemoteServiceEndpoint.java

```diff
@@ -79,7 +80,19 @@ public void expectMessages(List<String> messages) {
         messages.forEach(m -> listener
             .register(
                 post(urlEqualTo(path))
-                .willReturn(aResponse().withStatus(returnedStatusCode).withFixedDelay(delay))));
+                .willReturn(aResponse().withStatus(returnedStatusCode).withFixedDelay(delay).withHeader("Location", "http://localhost:1234"))));
```

Comment:
is the Location header required here?

Reply:
Actually no, it’s not required, removed it.